src/CommitmentsV2/SFA.DAS.CommitmentsV2.Messages/Events/ApprenticeshipCompletionDateUpdatedEvent .cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace SFA.DAS.CommitmentsV2.Messages.Events
+{
+    public class ApprenticeshipCompletionDateUpdatedEvent
+    {
+        public long  ApprenticeshipId { get; set; }
```

Comment:
My goodness, you could fit a lorry through the gap there.

Reply:
what gap?

Reply:
:-D The extra space between long and Apprenticeship 🚛